src/components/color/examples/ColorExamples.tsx

```diff
@@ -1,24 +1,41 @@
 import * as React from 'react';
 import * as _ from 'underscore';
-import {Tooltip} from '../../tooltip/Tooltip';
 import {Color} from '../Color';
 
 export class ColorExamples extends React.Component {
     render() {
         const VaporColors: {[key: string]: string} = require('!sass-variable-loader?preserveVariableNames!coveo-styleguide/scss/common/palette.scss');
         const colors = _.map(VaporColors, (value, color) => (

```

Comment:
We usually go with the following formatting when using ternary conditions, and don't forget to type the parameters or the mapping function with the right type (`: string`):
```jsx
const colors = _.map(VaporColors, (value: string, color: string) => condition
    ? (
        // result when condition is true
    )
    : (
        // result when condition is false
    )
);
```